src/include/txn_inline.h

```diff
@@ -514,8 +514,24 @@ __wt_txn_pinned_timestamp(WT_SESSION_IMPL *session, wt_timestamp_t *pinned_tsp)
 static inline bool
 __txn_visible_all_id(WT_SESSION_IMPL *session, uint64_t id)
 {
+    WT_TXN *txn;
     uint64_t oldest_id;
 
+    txn = session->txn;
+
+    /*
+     * When reading from a checkpoint, all readers use the same snapshot, so a transaction is
```

Comment:
The format of the comment is weird

Reply:
Odd. Investigating

Reply:
Curiously, if you just remove the spaces, clang-format sets it back that way. If you move some of the words around too, then it doesn't. Weird... anyway, fixed in [5bedb5e](https://github.com/wiredtiger/wiredtiger/pull/7946/commits/5bedb5e5645041ba501712e40a992eb385e67552)